udf/udf_js_json_extract_histogram.sql

```diff
@@ -0,0 +1,53 @@
+/*
+
+Returns a parsed struct from a JSON string representing a histogram.
+
+This implementation uses JavaScript and is provided for performance comparison;
+see udf_json_extract_histogram for a pure SQL implementation that will likely
+be more usable in practice.
+
+*/
+
+CREATE TEMP FUNCTION
+  udf_js_json_extract_histogram (input STRING)
```

Comment:
I think we should name this file `udf/udf_json_extract_histogram.js.sql` and name the function (for consistency with the file name):
```suggestion
  udf_json_extract_histogram_js (input STRING)
```

Reply:
if you disagree, it's also fine as-is, but we should update the readme accordingly